File: projects/store-or/src/test/java/com/robinfood/storeor/mocks/entity/configurationposbystore/StorePosEntityMock.java
package com.robinfood.storeor.mocks.entity.configurationposbystore;

import com.robinfood.storeor.entities.configurationposbystore.StorePosEntity;

import java.util.Collections;
import java.util.List;

public class StorePosEntityMock {

    public List<StorePosEntity> storePosEntities = List.of(
            StorePosEntity.builder()
                    .id(1L)
                    .name("caja1")
                    .resolutions(Collections.emptyList())
                    .build(),
            StorePosEntity.builder()
                    .id(2L)
                    .name("caja2")
                    .resolutions(Collections.emptyList())
                    .build()
    );
}

File: projects/posv2/repository/src/test/kotlin/com/robinfood/repository/mocks/posrelatedtoastore/StorePosEntityMock.kt
package com.robinfood.repository.mocks.posrelatedtoastore

import com.robinfood.core.entities.posrelatedtoastore.StorePosEntity

class StorePosEntityMock {

    val storePosEntities = listOf(
        StorePosEntity(
            1L,
            "pos 1",
            ResolutionEntityMock().resolutionEntity
        ),
        StorePosEntity(
            2L,
            "pos 2",
            ResolutionEntityMock().resolutionEntity
        )
    )
}
